### ISSUE TITLE
Crucibles don't seem to interact with most fluid transport pipes.

### ISSUE THREAD

**Issue by tacticalretard** [BloodWorkXGaming/ExNihiloCreatio#247]
I play in version 1.12.2 with exnihilocreatio-1.12.2-0.4, and Crucibles don't output their contained fluids into transport pipes such as Thermal Expansion Fluiducts, Actually Additions fluid transport relays, etc. Is this intended behavior or an issue? If it is an issue, is there a fix/workaround for it?

**Comment by SirLyle** [comment]
They don't automatically output fluids, you need something that will "pull" fluids. For TE ducts use a servo.

**Comment by SirLyle** [comment]
Could you please comment if that fixed your issue so that I can close this issue.

**Comment by tacticalretard** [comment]
Sorry for the late reply. Minecraft has been crashing for me for the past few days, so I've been trying to figure out what is wrong. That said, I had already tried using a thermal expansion fluiduct with a resonant servo before and it never seemed to extract the lava. When I figure out what has minecraft crashing, I'll try to test it again with different fluiducts and servos.

**Comment by tacticalretard** [comment]
Okay, I managed to fix my constant game crashes (which were unrelated to this mod), and I went back to test and resolve my issue. I tested out with different fluiducts and a combination of both retrievers and servos. It did indeed work in all instances, so the original issue was a problem on my part. I went back to the previous world I tested in and it turns out I had messed up the filters, which is what caused the issue in the first place. Apologies for the inconvenience.
